tutorials/fundamentals/parameters.md

```diff
@@ -0,0 +1,9 @@
+JavaScript, by default, does not support named parameters.
+However, you can force it to by using object literals and destructuring.
```

Comment:
Instead of "force it", I'd say "However, you can do something similar using object literals and destructuring."